chrome/content/zotero/zoteroPane.js

```diff
@@ -1016,6 +993,38 @@ var ZoteroPane = new function()
 					return;
 			}
 		}
+	}
+	
+	/*
+	 * Trigger actions based on keyboard shortcuts (non-capturing). This should be the default
```

Comment:
The "capturing" event handler is really poorly named by the designers of JS, given that "we capture an event" colloquially in any non-JS context would be understood as not allowing the event to propagate further. The comment for captureKeyDown() says "A capturing listener to handle", so this should say "A bubbling listener to handle..." and explain the use case - should be used when a lower-level element may override the navigation handling by stopping event propagation.

Reply:
That makes sense, thank you. I re-pushed with a new comment and an example to make it clearer:

	 * Bubbling listener for navigation or shortcuts keydown events that should be
	 * handled only if no lower-level element overrode it by stopping event propagation.
	 * E.g. Escape when reader is opened refocuses the scrollable area of the reader. This should
	 * not happen if Escape was pressed when a menupopup is opened - just let menupopup handle the
	 * Escape and close the popup.